src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -316,20 +263,7 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
         height: menuItemsRef?.current?.offsetHeight || 0,
       });
     }
-  }, []);
-
-  useEffect(() => {
-    if (hasSignedToday) {
-      newStorageService
-        .hasUploadedFiles()
-        .then(({ hasUploadedFiles }) => {
-          setHasAnyUploadedFile(hasUploadedFiles);
-        })
-        .catch((error) => {
-          errorService.reportError(error);
-        });
-    }
-  }, []);
+  }, [dimensions.height, dimensions.width]);
```

Comment:
I think this could lead to an infinite loop, couldn't it? Since the variables with the potential to be updated within the effect are dependencies